### ISSUE TITLE
"Some polygons of HeadMale not assigned to any body part." when exporting unmodified Fallout 3 model

### ISSUE THREAD

**Issue by LandFunco** [niftools/blender_niftools_addon#603]
@niftools/blender-niftools-addon-reviewer -

Before creating a new issue, ensure that
 - [X] Read the user documentation.
 - [X] Check that the issue hasn't already been reported.
 - [X] Read through the list of `Helpful resources` links on the right-hand side.
 - [ ] Talk with the developers to ensure its a valid issue

The issue will get closed off immediately as invalid if
 - [ ] Looking for support where the forums, chat or documentation is more appropriate.
 - [ ] Goes off topic, loses focus or turns into a discussion.
 - [ ] Removing parts of the template the issue.

Fill out the template below to the best of your ability, including .blend files, nifs, logs; help us to help you.

# Issue Overview
When trying to modify the default Fallout 3 head mesh I am unable to export the mesh due to the error:

```
Some polygons of HeadMale not assigned to any body part.The unassigned polygons have been selected in the mesh so they can easily be identified.
C:\Users\Me\AppData\Roaming\Blender Foundation\Blender\3.6\scripts\addons\io_scene_niftools\modules\nif_export\geometry\mesh\__init__.py:703
``` 

I also tried to import the default Fallout 3 head mesh and export it without any modifications, the same error occurs. All faces are highlighted after the export fails, but all of the faces are already assigned to vertex groups.

### Blender Niftools Addon Version Info
Blender Niftools Addon - v0.1.0

### Blender Version Info
Blender 3.6.0

### Platform information
Windows 10

## Context
Context is attempting to modify default head mesh for Fallout 3, I'm really just trying to zero out the shape keys in the EGM but I can't even get the unmodified mesh to re-export.

## Steps to Reproduce
1. Extract headhuman.nif from Fallout - Meshes.bsa
2. Import into Blender with default settings.
3. Export model as NIF with default settings.

## Expected Result
Model exports to NIF succesfully.

## Actual Result
The above error appears and already assigned vertices are highlighted. No NIF is created.

## Possible Fix
I don't know.

## Screenshot
![niftoolserror](https://github.com/niftools/blender_niftools_addon/assets/76762548/15fe8316-5ae8-4a9b-a966-d95f9b650aa5)

### Info Bar Output
[Blender Info Log.txt](https://github.com/niftools/blender_niftools_addon/files/12456817/Blender.Info.Log.txt)

### Console Output
[Blender console Log.txt](https://github.com/niftools/blender_niftools_addon/files/12456831/Blender.console.Log.txt)

### Blend File
[imported.zip](https://github.com/niftools/blender_niftools_addon/files/12456841/imported.zip)

### Nif File
[headhuman.zip](https://github.com/niftools/blender_niftools_addon/files/12456848/headhuman.zip)

## Similar Known Issues
https://github.com/niftools/blender_niftools_addon/issues/496

## Additional Information
N/A

**Comment by mirkiss** [comment]
Some polygons of HeadMale not assigned to any body part.The unassigned polygons have been selected in the mesh so they can easily be identified.

**Comment by mirkiss** [comment]
Exporting FXB is possible

**Comment by Candoran2** [comment]
It seems to be an import problem. Not exactly sure what caused it, but it seems to have been fixed after that, at least in my fork (#591).

Also, do be aware that for body parts, it's not _vertex_ groups that need to be assigned. Instead, it's _face_ maps:
![image](https://github.com/niftools/blender_niftools_addon/assets/45334438/6da40345-47bb-433f-af8e-ea281c919b80)

**Comment by Candoran2** [comment]
New version has been released.
